src/app/components/full-notes/notes-list/notes-list.component.ts

```diff
@@ -46,8 +47,12 @@ export class NotesListComponent implements OnInit {
 
   ngOnInit() {
     const isDisplayType$ = this.store.selectDisplayType$;
-    const notes$ = this.store.selectFilteredNotes$;
-    this.data$ = combineLatest([notes$, isDisplayType$]).pipe(
+    const filteredNotes$ = this.store.selectFilteredNotes$;
```

Comment:
redundant const. `this.filteredData$ = combineLatest([this.store.selectFilteredNotes$, isDisplayType$])`